WBooks/en.lproj/Localizable.strings

```diff
@@ -0,0 +1,11 @@
+/*
+  Localizable.strings
+  WBooks
+
+  Created by Juan Silva on 09/06/2021.
+  
+*/
+
+/* Titles views */
+"LIBRARY_TITLE" = "LIBRARY";
+
```

Comment:
Looks great! Let's add the strings from the Login screen as well ("user", "password", "Ingresar", etc).